addons/website_sale_collect_wishlist/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': "Product Availability Notifications",
+    'category': 'Website/Website',
+    'summary': "Bridge module for Website sale collect and wishlist",
+    'description': """
+It allows adding products to the wishlist when pickup location is not found
```

Comment:
But the warning display is not because of this module, this module works just when we want to add product to wishlist if product is out of stock :thinking:

Reply:
`It allows adding products to the wishlist when pickup location is not found`
Do we add product in wishlist when pickup location is not found? :thinking:

Reply:
Maybe we can add something like 
_"Enables users to add products to their wishlist when the selected pickup location is out of stock"_

Reply:
Yes sure! (or something generic but it's okay for now)